<launch>

  <!-- launch nodes  -->
  <node name="left_wheels" pkg="rover" type="left_wheels.py" output="screen"/>
  <node name="right_wheels" pkg="rover" type="right_wheels.py" output="screen"/>

</launch>
